docker/openemr/7.0.3/openemr.sh

```diff
@@ -451,6 +451,8 @@ echo " > https://opencollective.com/openemr/donate"
 echo ""
 
 if [ "$OPERATOR" == "yes" ]; then
+    echo "Starting php-fpm!"
+    /usr/sbin/php-fpm
```

Comment:
![off-to-the-races](https://github.com/openemr/openemr-devops/assets/5618372/89f6ab2f-946f-41fc-bbc5-5259fd7ba77c)